README.md

```diff
@@ -3,44 +3,116 @@ Some helpful snippets of code to automate the creation of Cluster-Stacks
 
 ## Goals
 Creating cluster stacks and cluster based on these takes a significant
-number of stacks that are hard to remember correctly for people that are
-not cluster-API and Cluster Class expert.
+number of steps that are hard to remember correctly for people that are
+not cluster-API and Cluster Class experts.
 
-We this hide them in a number of distinct steps that are in numbered scripts.
-The reason for not doing everything in one script is that you do register
-cloud secrets or install capi much less often than install cluster classses
-which happens much less often than creating clusters.
+We hide them in a number of distinct scripts. The reason for not doing
+everything in one script is that you register cloud secrets or install CAPI
+much less often than install cluster classes, which happens much less often
+than creating clusters.
+
+## Prerequisites and Setup
+
+### Obtaining the Cloud-in-a-Box Certificate
+
+If you're using SCS Cloud-in-a-Box (CiaB), you'll need to obtain the certificate. The certificates are located on the CiaB manager host at `/etc/ssl/certs/ca-certificates.crt`.
```

Comment:
```suggestion
If you're using SCS Cloud-in-a-Box (CiaB), you'll need to obtain the certificate. The certificates are located on the CiaB manager host at `/etc/ssl/certs/ca-certificates.crt`.
This step is only required because CiaB uses self-signed certificates -- please skip it on clouds that use TLS certificates that are signed by a proper certificate authority (CA).
```